test/hack/resource/pkg/resourcetypes/oidc.go

```diff
@@ -47,6 +49,9 @@ func (o *OIDC) GetExpired(ctx context.Context, expirationTime time.Time) (names
 
 	errs := make([]error, len(out.OpenIDConnectProviderList))
 	for i := range out.OpenIDConnectProviderList {
+		if o.Global() && !strings.Contains(lo.FromPtr(out.OpenIDConnectProviderList[i].Arn), lo.Must(config.LoadDefaultConfig(ctx)).Region) {
```

Comment:
What is this for? Don't we know that this resource is global so why do we need to check it within the method?

Reply:
We will have a case were a sweeper running for  us-west-2 can clean up resources meant for eu-north-1 as those resource will be discovered as well.